clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/VeniceChangeCoordinate.java

```diff
@@ -67,7 +68,7 @@ public int comparePosition(VeniceChangeCoordinate other) {
       // but it DOESNT account for version rollbacks.
       return topic.compareTo(other.topic);
     }
-    return pubSubPosition.comparePosition(other.pubSubPosition);
+    return PubSubUtil.comparePubSubPositions(pubSubPosition, other.pubSubPosition);
```

Comment:
actually i did not get the reasoning from the PR description why we want to compare position through this static method temporarily before this operation is done by TopicManager.

Reply:
can you explain "This simplifies the interface and removes assumptions about position comparison logic from the position abstraction itself." in more detail?